src/Components/Form_register/Form_Register.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react'
+import './Form_Reg.css'
+class Form_Register extends React.Component{
+
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            Name: '',
+            Email: '',
+            PhoneNumber:'',
+            Password: '',
+            Avatar: '',
+            Gender: 'Male',
+            Age: 1,
+            Note: '',
+            Favorite: []
+            }
+        }
+
+    onChangeFavorite = (value) => {
+        let New_Favorite = Array.from(this.state.Favorite)
+        if( New_Favorite.indexOf(value) === -1){
+            New_Favorite.push(value)
+            console.log('New_Favorite',New_Favorite)
+            console.log('Add success!' ,value)
+        }else{
+            let index = New_Favorite.findIndex( item => item === value )
+            New_Favorite.splice(index,1)
+            this.setState({Favorite: New_Favorite})
+            console.log('Remove success!' ,value)
+            console.log('New_Favorite',New_Favorite)
+        }
+        this.setState({Favorite: New_Favorite})
+    }
+
+    onChangeGenger = (value) => {
+        this.setState({Gender: value})
+    }
+
+    HandleChangeInput = (event) => {
+        let Value = event.target.value
+        let Name = event.target.name
+        this.setState({ [Name]: Value })
+    }
+
+    onSubmit = () => {
+        console.log(this.state)
+    }
+
+    render() {
+
+        return (
+        <div className="form_register">
+            <h1>Đăng kí thành viên</h1>
+            <form>
+            <table border={1}>
+
+                <thead>
+                <tr>
+                    <th>
+                    Nội dung
+                    </th>
```

Comment:
Viết lên thành 1 dòng
```suggestion
                    <th>Nội dung</th>
```